src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -151,5 +221,88 @@ public Task<TagHelperContent> GetChildContentAsync(bool useCachedResult)
         {
             return _getChildContentAsync(useCachedResult);
         }
+
+        /// <inheritdoc />
+        public void WriteTo(TextWriter writer, HtmlEncoder encoder)
```

Comment:
This entire method is copied from MVC `WriteTagHelperToAsync` with the exception of [this](https://github.com/aspnet/Razor/pull/615/files#r45275825).